logging/Differences_between_ODFE_and_OpenSearch.md

```diff
@@ -0,0 +1,97 @@
+
+The OpenSearch deployment script does not do this automatically. If you want to make OpenSearch Dashboards accessible via NodePort, you must set the environment variable `KB_KNOWN_NODEPORT_ENABLE` to `True` before running the `logging/bin/deploy_logging_opensearch.sh` script.
+
+Kibana was accessible from port 31033. OpenSearch Dashboards is accessible from port 31034.
+
+**Note:** NodePorts are not suitable for production deployments. NodePorts are intended only as a convenience for internal testing groups working in an OpenStack environment. 
+
+### Migrate an Existing Deployment
+
+To migrate an existing deployment of the log monitoring project, run the `./logging/bin/deploy_logging_opensearch.sh` script.
+If an existing deployment is detected in the namespace, that deployment
+is migrated to the OpenSearch-based deployment.
+
+### Remove OpenSearch-based Log Monitoring
+
+To remove log monitoring that uses OpenSearch instead of ODFE, run the `logging/bin/remove_logging_opensearch.sh` script. The new script supports all the same environment variables as the prior script.
+
+### Running Other Scripts
+
+Other than the scripts described in the [table](#compiled_dif_table), the remaining log monitoring  scripts work with either search back-end. 
+
+See [Compiled Differences](#compiled_dif_table) for the list of changed script file names.
+
+**Important:** During the experimental phase only, you must set the environment variable `LOG_SEARCH_BACKEND` to `OPENSEARCH` (in uppercase) when you run scripts with OpenSearch.
+
+### <a name="compiled_dif_table"></a>Compiled Differences
+
+Stable Release 1.1.8 and Earlier (ODFE) | OpenSearch Experimental Release 1.1.8  | OpenSearch Production Release 1.2.0
+----|----|----
+**Script File Name Changes between Releases**
+logging/bin/deploy_logging_open.sh | logging/bin/deploy_logging_opensearch.sh | logging/bin/deploy_logging.sh 
+logging/bin/deploy_logging_open_openshift.sh | logging/bin/deploy_logging_opensearch_openshift.sh |	logging/bin/deploy_logging_openshift.sh
+logging/bin/remove_logging_open.sh | logging/bin/remove_logging_opensearch.sh | logging/bin/remove_logging.sh
+logging/bin/remove_logging_open_openshift.sh | logging/bin/remove_logging_opensearch_openshift.sh | logging/bin/remove_logging_openshift.sh
+**YAML File Name Changes between Releases** **Note:** The OpenSearch YAML files required format changes from the Elasticsearch YAML file. See samples/ingress or samples/tls for more details. | |
+$USER_DIR/logging/user-values-elasticsearch-open.yaml (Elasticsearch configuration changes) | $USER_DIR/user-values-elasticsearch-opensearch.yaml | $USER_DIR/logging/user-values-opensearch.yaml
+$USER_DIR/logging/user-values-elasticsearch-open.yaml (Kibana configuration changes) | 	$USER_DIR/user-values-osd-opensearch.yaml | $USER_DIR/logging/user-values-osd.yaml
+**Kubernetes Resource Name Changes between Releases** ||
```

Comment:
I suggest we clarify the type of Kubernetes resources in each row.

Reply:
Changed made.